ESPLoader.js

```diff
@@ -777,7 +777,10 @@ class ESPLoader {
         let image, address;
         for (var i = 0; i < fileArray.length; i++) {
             console.log("Data Length " + fileArray[i].data.length);
-            image = fileArray[i].data + '\xff\xff\xff\xff'.substring(0, 4 - fileArray[i].data.length % 4);
+
+            image = fileArray[i].data;
+            const reminder = fileArray[i].data.length % 4;
```

Comment:
Changes look good. Minor typo suggestion - can we call it remainder instead of reminder?